graphql/graphql.go

```diff
@@ -910,9 +910,9 @@ func (p *Pending) Call(ctx context.Context, args struct {
 	if err != nil {
 		return nil, err
 	}
-	status := hexutil.Uint64(1)
+	status := Long(1)
 	if result.Failed() {
-		status = 0
+		status = Long(0) // TODO is this necessary
```

Comment:
This should suffice (otherwise the compiler will yell at you): 

```suggestion
		status = 0
```
it's already defined as `Long`, so I don't think you need an explicit cast